src/libraries/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/Marshal/OffsetOfTests.cs

```diff
@@ -32,9 +33,14 @@ public void OffsetOf_ClassWithSequentialLayout_ReturnsExpected()
             Assert.Equal(new IntPtr(4), Marshal.OffsetOf(typeof(MyPoint), nameof(MyPoint.y)));
         }
 
-        [Fact]
+        [ConditionalFact]
         public void OffsetOf_ExplicitLayout_ReturnsExpected()
         {
+            if (OperatingSystem.IsAndroid() && RuntimeInformation.ProcessArchitecture == Architecture.X86)
+            {
+                throw new SkipTestException("https://github.com/dotnet/runtime/issues/49872 Test fails on Android x86 with -- Expected: 56, Actual: 52");
+            }
+
```

Comment:
I know I suggested this but I just remembered a better option that preserves the `ActiveIssue` (so we don't forget it):

```suggestion
        [Fact]
        [ActiveIssue("https://github.com/dotnet/runtime/issues/49872", typeof(PlatformDetection), nameof(PlatformDetection.IsAndroid), nameof(PlatformDetection.Is32BitProcess)]
        public void OffsetOf_ExplicitLayout_ReturnsExpected()
        {
```